src/elements/Order.php

```diff
@@ -2316,37 +2298,37 @@ public function getUser(): ?User
      *
      * @param string|null $email
      * @throws Exception
+     * @deprecated in 4.3.0. Use [[setCustomer()]] instead.
      */
     public function setEmail(?string $email): void
     {
+        Craft::$app->getDeprecator()->log(__METHOD__, '`Order::setEmail()` has been deprecated use `Order::setCustomer()` instead.');
         if (!$email) {
             $this->_customer = null;
             $this->_customerId = null;
-            $this->_email = null;
```

Comment:
any reason not to clear the `this->_email` ?

Reply:
Where have you seen that, I can't see it in the code?